Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
+#   2.  Omi loglevel must be INFO or higher
+#   3.  Current user needs to be enabled for sudo
```

Comment:
You can test this too with something like `sudo /bin/true` and be sure the exit code is zero.

But if the user is currently root, then you don't need this, and SUDO variable should be set to empty (i.e. $SUDO would be ignored). And above message should read `Current user needs to be enabled for sudo, or current user must be running as root user`